contrib/backporting/submit-backport

```diff
@@ -18,6 +18,14 @@ DIR=$(dirname $(readlink -ne $BASH_SOURCE))
 source $DIR/../release/lib/common.sh
 source $DIR/common.sh
 
+require_linux
```

Comment:
I believe this line has been pulled in by accident by conflict resolution. It's not available on 1.7, so the script will crash. 

We should probably simply backport the underlying commit: https://github.com/cilium/cilium/commit/05eabc71a8525fffba8977eabdfd6e763dcd08c7 (not the full PR)